context/app/static/js/components/DatasetDetail.jsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import styled from 'styled-components';
+import Container from '@material-ui/core/Container';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import { Vitessce } from 'vitessce';
+import VisTabs from './VisTabs';
+import DetailSummary from './DetailSummary';
+import DetailAttribution from './DetailAttribution';
+import DetailProtocol from './DetailProtocols';
+import NoticeAlert from './NoticeAlert';
+import DetailMetadataTable from './DetailMetadataTable';
+import DetailFileTable from './DetailFileTable';
+import 'vitessce/build-lib/es/production/static/css/index.css';
+
+const FlexContainer = styled(Container)`
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+`;
+
+const SpacedContainer = styled(FlexContainer)`
+    justify-content: space-evenly;
+`;
+
+const StyledDivider = styled(Divider)`
+  margin-left: 5px;
+  margin-right: 5px;
+  height: 15px;
+  background-color: #444A65;
+  align-self:center;
+`;
+
+
+function AssaySpecificItem(props) {
+  const { children } = props;
+  return (
+    <>
+      <Typography variant="body1">{children}</Typography>
+      <StyledDivider orientation="vertical" flexItem />
+    </>
+  );
+}
+
+function SummaryData(props) {
+  const { data_types, origin_sample } = props;
+  return (
+    <>
+      {data_types && data_types.length
+        ? (<AssaySpecificItem>{data_types.constructor.name === 'Array' ? data_types.join(' / ') : data_types}</AssaySpecificItem>) : null}
+      {origin_sample.organ && origin_sample.organ.length
+        ? (<Typography variant="body1">{origin_sample.organ}</Typography>) : null}
```

Comment:
Here and below, could `&&` be used here instead of `?:`?

Reply:
I'll look into this further when time is available. Originally I was using && predominantly, but it was rendering a 0 in some instances where the test evaluated false. I opted to use the ternary which I know works until I can figure everything out.

Reply:
FWIW This is why I often go for ternary instead as well - tighter control.